src/selectors/getErrors.ts

```diff
@@ -24,6 +24,7 @@ const getHasErrorsSelector = (state: IRuntimeState) => {
     state.profile.error ||
     state.language.error ||
     state.formLayout.error ||
+    state.formLayout.error ||
```

Comment:
It should probably be the footer layout state error, not the form layout error twice. :grin:

Reply:
Wow 🤦 nice catch

Reply:
looks like my typescript extension bugged out and could not find the file, so the autocomplete messed me up 😅